.github/workflows/ci.yaml

```diff
@@ -76,6 +76,21 @@ jobs:
           fetch-depth: 0
           submodules: true
 
+      # Hack alert: actions/checkout will check out the (disjunct) merge commit of a PR
+      #  instead of its head commit. That commit is not connected to any branch.
+      # This causes breakage downstream e.g. when the devcontainer test wants to check out
+      #  the ref in the scripts repo that corresponds to this build.
+      - name: Check out head commit of PR
```

Comment:
I think we should also do this in the kola workflow for consistency - otherwise it's confusing if one works on the merged state and the other not.

Reply:
Oh right! Good catch.